### PR TITLE
check for solr.jetty.https.port porp to set https urlscheme

### PR DESCRIPTION
**Pull request by hiteshk25**

### CODE REVIEW

File: solr/core/src/java/org/apache/solr/cloud/ZkController.java [hunk 1]
```diff
@@ -407,7 +408,11 @@ public ZkController(
 
       // note: Can't read cluster properties until createClusterState ^ is called
       final String urlSchemeFromClusterProp =
-          zkStateReader.getClusterProperty(ZkStateReader.URL_SCHEME, ZkStateReader.HTTP);
+          zkStateReader.getClusterProperty(
```
Comment (hiteshk25) [inline solr/core/src/java/org/apache/solr/cloud/ZkController.java]: When SSL is enabled we set this java sys prop `"solr.jetty.https.port"`(ie HTTPS_PORT_PROP)

**Review by justinrsweeney — APPROVED** [review]
LGTM

File: solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/ZkStateReader.java [hunk 2]
```diff
@@ -1149,6 +1150,12 @@ private void loadClusterProperties() {
               ClusterProperties.convertCollectionDefaultsToNestedFormat(properties);
           log.debug("Loaded cluster properties: {}", this.clusterProperties);
 
+          if (!this.clusterProperties.containsKey(URL_SCHEME)) {
```
Comment (hiteshk25) [inline solr/solrj-zookeeper/src/java/org/apache/solr/common/cloud/ZkStateReader.java:1153]: @justinrsweeney solr reads cluster properties from multiple places (URL_SCHEME). Thus adding here that check, when solr reads cluster-props file from zk. Let me know if this looks reasonable

**Comment by hiteshk25** [comment]
It seems clusterprops has very sophisticated mechanism in solr. Given the complexity its not worth. Closing this PR/ticket.

**Pull request closed without merging**